src/Apps/IdentityVerification/IdentityVerificationApp/CompleteFailed.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Button, Link, Sans, Serif } from "@artsy/palette"
+import React from "react"
+
+export const CompleteFailed: React.FC = () => {
+  return (
+    <Box px={[2, 3]} mb={6} mt={4}>
+      <Box
+        mx={["auto"]}
```

Comment:
```suggestion
        mx="auto"
```

Reply:
Saw that this was resolvrd, but @yuki24's requested changes wern't applied.

Reply:
yes, i have them all in a working branch, sorry - will change pr to WIP